src/subcommands/crates.rs

```diff
@@ -27,22 +48,72 @@ pub fn crates(args: Vec<String>, max_age: std::time::Duration) -> Result<(), std
         publishers.sort_unstable_by_key(|p| (p.kind, p.login.clone()));
     }
 
-    println!("\nDependency crates with the people and teams that can publish them to crates.io:\n");
-    for (i, (crate_name, publishers)) in ordered_owners.iter().enumerate() {
-        let pretty_publishers: Vec<String> = publishers
-            .iter()
-            .map(|p| match p.kind {
-                PublisherKind::team => format!("team \"{}\"", p.login),
-                PublisherKind::user => p.login.to_string(),
-            })
-            .collect();
-        let publishers_list = comma_separated_list(&pretty_publishers);
-        println!("{}. {}: {}", i + 1, crate_name, publishers_list);
-    }
+    if output_json {
+        let mut jdoc = CratesJdoc{
+            warning: concat! (
+                "Note: there may be outstanding publisher invitations. crates.io provides no way to list them. ",
+                "See https://github.com/rust-lang/crates.io/issues/2868 for more info."
+            ).to_string(),
```

Comment:
Concat for a string shouldn't be necessary, you can use [an escape with multi-line strings](https://stackoverflow.com/questions/29483365/what-is-the-syntax-for-a-multiline-string-literal/29483453#29483453). I'm not sure if this should be formatted across two lines in the final output since it's probably not relevant for json output itself, only display on a terminal.